### ISSUE TITLE
[Bug]: After creating a collection using the BM25 function, unable to insert  data into this collection in a dataframe format.

### ISSUE THREAD

**Issue by zhuwenxing** [milvus-io/milvus#36299]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Environment

```markdown
- Milvus version:
- Deployment mode(standalone or cluster):
- MQ type(rocksmq, pulsar or kafka):    
- SDK version(e.g. pymilvus v2.0.0rc2):
- OS(Ubuntu or CentOS): 
- CPU/Memory: 
- GPU: 
- Others:
```

### Current Behavior

```
pymilvus.exceptions.ParamError: <ParamError: (code=number of fields: 5, number of entities: 8, message=)>
 (api_request.py:45)
[2024-09-14 18:01:46 - ERROR - ci_test]: (api_response) : <ParamError: (code=number of fields: 5, number of entities: 8, message=)> (api_request.py:46)
```
```

        prefix = "test_bm25_search_normal"
        analyzer_params = {
            "tokenizer": "default",
        }
        dim = 128
        default_fields = [
            FieldSchema(name="id", dtype=DataType.INT64, is_primary=True),
            FieldSchema(name="word", dtype=DataType.VARCHAR, max_length=65535, enable_match=True,
                        analyzer_params=analyzer_params),
            FieldSchema(name="title", dtype=DataType.VARCHAR, max_length=65535, enable_match=True,
                        analyzer_params=analyzer_params),
            FieldSchema(name="text", dtype=DataType.VARCHAR, max_length=65535, enable_match=True,
                        analyzer_params=analyzer_params),
            FieldSchema(name="word_sparse", dtype=DataType.SPARSE_FLOAT_VECTOR),
            FieldSchema(name="title_sparse", dtype=DataType.SPARSE_FLOAT_VECTOR),
            FieldSchema(name="text_sparse", dtype=DataType.SPARSE_FLOAT_VECTOR),
            FieldSchema(name="dense_emb", dtype=DataType.FLOAT_VECTOR, dim=dim)
        ]
        default_schema = CollectionSchema(fields=default_fields, description="test collection")
        for field in ["word", "title", "text"]:
            bm25_func = Function(
            name=f"bm25_{field}",
            function_type=FunctionType.BM25,
            inputs=[field],
            outputs=[f"{field}_sparse"],
            params={"bm25_k1": 1.2, "bm25_b": 0.75},
            )
            default_schema.add_function(bm25_func)
        print(f"\nCreate collection")

        collection_w = self.init_collection_wrap(name=cf.gen_unique_str(prefix), schema=default_schema)
        nb = 3000
        data = []
        for i in range(nb):
            d = {
                "id": i,
                "word": fake_en.word(),
                "title": fake_en.sentence(),
                "text":fake_en.paragraph(),
                "dense_emb": cf.gen_vectors(1, dim)[0]
            }
            data.append(d)
        df = pd.DataFrame(data)
        log.info(f"data\n{df}")
        collection_w.insert(df)
```

### Expected Behavior

When inserting data, ignore the automatically generated sparse columns.

**Comment by zhengbuqian** [comment]
related code has been merged into pymilvus repo, can you try again with latest version of pymilvus?

/assign @zhuwenxing

**Comment by zhuwenxing** [comment]
verified and fixed in pymilvus `2.5.0rc96`
